controllers/ucs_controller.py

```diff
@@ -228,6 +228,24 @@ def getServiceProfile():
 
 
 @http_body_factory
+def powerStatus(host=None, user=None, password=None, identifier=None):
```

Comment:
It looks like you are pulling the host/user from the header; Do you still need them here?